packages/astro/src/@types/astro.ts

```diff
@@ -1010,6 +1012,71 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		service: ImageServiceConfig;
+
+		/**
+		 * @docs
+		 * @name image.domains (Experimental)
+		 * @type {Array<string>}
+		 * @default `{domains: []}`
+		 * @version 2.9.2
```

Comment:
```suggestion
		 * @version 2.10.10
```
?? Check this! I'm guessing though that 2.9.2 is not what you intended.